sale css init

osadchyi15 opened the pull request.

osadchyi15 committed:
sale css init

osadchyi15 committed:
Merge branch 'main' into osadchyi-sale-css

Natatulichka commented:
.sale-descr-tags-list  font-size 12px
@media only screen and (min-width: 768px) {.sale-descr-tags-list  font-size 14px

osadchyi15 committed:
Merge branch 'main' into osadchyi-sale-css

osadchyi15 commented:
Подивись, будь ласка, правила
.sale-descr-tags-LINK
для обох медіазапитів. Там вказані потрібні розміри шрифтів. Вони вказані саме для посилання.

osadchyi15 committed:
add comments

osadchyi15 committed:
change tag a (text to link) + actual price

osadchyi15 committed:
change tag a (text to link) + actual price

osadchyi15 committed:
Merge branch 'osadchyi-sale-css' of https://github.com/Natatulichka/DDT into osadchyi-sale-css

osadchyi15 committed:
Merge branch 'main' into osadchyi-sale-css

osadchyi15 committed:
add classes for images, restore wrapper

osadchyi15 committed:
rubber layout

Наташа, подивись дуже уважно. Я зробив гумову верстку для sale. Вірніше тільки для частини з зображеннями, щоб розміри блоків, та зображень у них адекватно змінювали розмір.

Для цього я змінив padding для контейнера від 320 до 375 px у reset.css.

Padding буде змінюватися при зменшені вьюпорта для всіх секцій, але не має їх ламати, якщо у колег вірно написано.

Від 375px padding фіксований - 20px

osadchyi15 committed:
add id for section

osadchyi15 committed:
tags text position

.sale-descr-tags-link {
display: block;
text-align: center;
vertical-align: middle;
